.openpublishing.redirection.json

```
@@ -6929,6 +6929,11 @@
       "source_path": "exchange/virtual-folder/exchange/Set-DefaultTenantBriefingConfig.md",
       "redirect_url": "/viva/insights/personal/reference/briefing-pause",
       "redirect_document_id": false
+    },
+    {
+      "source_path": "teams/virtual-folder/teams/New-CsTeamsShiftsConnectionTeamMap.md",
+      "redirect_url": "powershell/module/teams",
```

```suggestion
      "redirect_url": "/powershell/module/teams",
```
Site relative links must start with a slash